.pre-commit-config.yaml

```diff
@@ -408,13 +408,25 @@ repos:
         language: pygrep
         name: Check for language that we do not accept as community
         description: Please use "deny_list" or "allow_list"  instead.
-        entry: "(?i)(black|white)[_-]?list"
+        entry: >
+          (?ix)
+          \bmaster\b|
+          \bhe\b|
+          \bshe\b|
+          \bhis\b|
+          \bher\b|
+          \bslave\b|
+          \bsanity\b|
+          \bdummy\b|
+          \bwhite\b|
+          \bblack\b
```

Comment:
maybe we should stay with looking for black/white list.
It will be very difficult to handle colours names without excluding many files